awesome.js

```diff
@@ -65,7 +65,14 @@ class Awesome{
             this,
             {
                 /**
-                 * Path to folder awesome.js is located in
+                 * Path to folder awesome.js is located in.
+                 *
+                 * @example
+                 *
+                 * //use awesome.path to require scripts
+                 * awesome.requireCSS(`${awesome.path}components/buttons/awesome-buttonset.css`);
+                 * awesome.requireScript(`${awesome.path}components/buttons/awesome-buttonset.js`);
+                 *
```

Comment:
You require more than just scripts with this and can use ot for many other reasons.

The following would be better.

```javascript

//use awesome.path to refrence the awesome-webcomponents directory

```